backend/corpora/dataset_processing/process_download_validate.py

```diff
@@ -54,6 +54,9 @@ def process(dataset_id: str, dropbox_url: str, artifact_bucket: str):
 
     # Upload the labeled dataset to the artifact bucket
     bucket_prefix = get_bucket_prefix(dataset_id)
+    create_artifact(
```

Comment:
nit: mind adding a comment (or adding to the comment right above) to explain that we're saving both the unlabeled and labeled h5ads?